CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # PayPal iOS SDK Release Notes
 
+## Unreleased
```

Comment:
nit: [CI looks for lowercased unreleased](https://github.com/paypal/paypal-ios/blob/2e7393cd4d2a93c710d6f95b3ea359c3d1608270/.github/workflows/release.yml#L20):
```suggestion
## unreleased
```